fiftyone/server/routes/camera.py

```
@@ -7,19 +7,174 @@
 """
 
 import logging
-from typing import List, Optional
+from typing import Dict, Generator, List, Optional, Set
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)